src/Model/RestAPIMethod.cs

```diff
@@ -154,6 +154,8 @@ public void AddImportsTo(ISet<string> imports, bool includeImplementationImports
 
             if (includeImplementationImports)
             {
+                imports.Add("io.reactivex.functions.Function");
```

Comment:
Please add an entry in the ClassType.cs file for the Function type and then use the new entry here. That makes it much easier to determine usages of Function in the code.